helix-rest/src/main/java/org/apache/helix/rest/server/resources/zookeeper/ZooKeeperAccessor.java

```diff
@@ -94,6 +96,8 @@ public Response get(@PathParam("path") String path, @QueryParam("command") Strin
         return getChildren(_zkBaseDataAccessor, path);
       case getStat:
         return getStat(_zkBaseDataAccessor, path);
+      case delete:
```

Comment:
Would this go to a "DELETE" HTTP verb?

Reply:
That makes more sense, let me have a try.